cpp/src/parquet/file_reader.cc

```diff
@@ -272,7 +273,7 @@ class SerializedRowGroup : public RowGroupReader::Contents {
   std::unique_ptr<RowGroupMetaData> row_group_metadata_;
   ReaderProperties properties_;
   int row_group_ordinal_;
-  const std::unordered_set<int> prebuffered_column_chunks_;
+  std::shared_ptr<Buffer> prebuffered_column_chunks_bitmap_;
```

Comment:
Why not this a `const std::shared_ptr<Buffer> prebuffered_column_chunks_bitmap_;`?